### ISSUE TITLE
Ufo Decorator Table Index Is Nil

### ISSUE THREAD

**Issue by darmawan01** [kevinhwang91/nvim-ufo#201]
### Neovim version (nvim -v | head -n1)

NVIM v0.9.1

### Operating system/version

WSL 2 - Terminal

### How to reproduce the issue

Lazyvim config:

```bash
return {
  {
    "kevinhwang91/nvim-ufo",
    event = "BufRead",
    dependencies = {
      { "kevinhwang91/promise-async" },
      {
        "luukvbaal/statuscol.nvim",
        config = function()
          local builtin = require("statuscol.builtin")
          require("statuscol").setup({
            -- foldfunc = "builtin",
            -- setopt = true,
            relculright = true,
            segments = {
              { text = { builtin.foldfunc }, click = "v:lua.ScFa" },
              { text = { "%s" }, click = "v:lua.ScSa" },
              { text = { builtin.lnumfunc, " " }, click = "v:lua.ScLa" },
            },
          })
        end,
      },
    },
    config = function()
      -- Fold options
      vim.o.fillchars = [[eob: ,fold: ,foldopen:,foldsep: ,foldclose:]]
      vim.o.foldcolumn = "1" -- '0' is not bad
      vim.o.foldlevel = 99 -- Using ufo provider need a large value, feel free to decrease the value
      vim.o.foldlevelstart = 99
      vim.o.foldenable = true

      require("ufo").setup()
    end,
  },
}
```

### Expected behavior

No Error

### Actual behavior

Got an error when open nvim

![image](https://github.com/kevinhwang91/nvim-ufo/assets/23370707/20df68c0-8e1a-4a2d-a9b5-11b56d1676e0)

**Comment by darmawan01** [comment]
I was followed this config https://github.com/kevinhwang91/nvim-ufo/issues/117#issuecomment-1465111276

**Comment by darmawan01** [comment]
Solved, use the latest of Neovim 0.9.5
